src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+
+function getUser(userId: number): User | null {
```

Comment:
in basic naming convention `getUser` means that error should be thrown if user is not found.
In this case it's better to use `findUser`.
```suggestion
function findUser(userId: number): User | null {
```